NEWS.md

```diff
@@ -28,6 +28,7 @@
 
 ## New and improved features
 
+* All user-facing messages are now prepared using the [`{cli}`](https://github.com/r-lib/cli) package (#2418, @IndrajeetPatil). All messages have been reviewed and updated to be more informative and consistent.
```

Comment:
I thought I remembered {pkgdown} automatically linking packages like `{pkg}`, but the current NEWS does not do so, any idea?

Reply:
Oh, {pkgdown} does link it, but I had more in mind someone who might be reading the NEWS on CRAN.

But, on a second thought, maybe it's better to leave out the URL since it's not that hard to someone just look for cli on CRAN.